src/beignet/_maximum_mean_discrepancy.py

```diff
@@ -0,0 +1,89 @@
+import numpy
```

Comment:
probably better to use torch instead of numpy for consistency unless numpy is needed for a particular reason

Reply:
would be nice if this worked with arrays of strings as well since that's a common data structure. We allowed numpy arrays here for the same reason: https://github.com/Genentech/beignet/blob/main/src/beignet/_farthest_first_traversal.py

Reply:
that being said I could see wanting this both differentiable and GPU-enabled... any thoughts on that? implement two versions?

Reply:
casting to tensor opens the tokenization and padding can of worms and usually you just want something simple